README.md

```diff
@@ -405,9 +405,45 @@ class Run(BaseModel):
 
 </p></details>
 
+### Example with preamble
+
+<details><summary>----- Show -----</summary>
+<p>
+A simple example to demonstrate adding extra code before the class list.
+
+```sh
+json2models -f pydantic --preamble "# set up defaults
+USERNAME = 'user'
+SERVER_IP = '127.0.0.1'
+" -m Swagger testing_tools/swagger.json
```

Comment:
not very easy to show multi-line arguments with the current readme structure so I added this extra example